internal/clients/http/ec2/ec2_client.go

```diff
@@ -358,3 +365,14 @@ func (c *ec2Client) parseRunInstancesResponse(respAWS *ec2.RunInstancesOutput) [
 	}
 	return list
 }
+
+func (c *ec2Client) GetAccountId(ctx context.Context) (string, error) {
+	input := &sts.GetCallerIdentityInput{}
+
+	out, err := c.sts.GetCallerIdentity(ctx, input)
```

Comment:
if I understood correctly this endpoint is being added in order to receive the provisioning app account ID? 
Can you please add an explanation on why is it needed for your service?
(it will be great if written in the description of the PR) I wasn't aware of it

Reply:
Updated the PR description, hope it's enough details.